services/image-analysis/.gitignore

```diff
@@ -0,0 +1,33 @@
+HELP.md
```

Comment:
We already have a .gitignore file at the top of the repository. Could you remove this one and add the Java related ignores in that file?

Reply:
Also, please add .DS_Store to .gitignore too and remove them from the checkin